src/components/footer/Footer.jsx

```diff
@@ -1,35 +1,79 @@
 import React from 'react';
 import classes from './Footer.module.scss';
-import { privacyNotice } from '../../assets';
+// import { nowUOrange } from '../../assets';
```

Comment:
And similarly to #58 again, I think to keep things clean let's remove any unneeded code where possible